### PR TITLE
[MU4] Restructure project - removed scene and domain sub dirs

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: mu4/notation/CMakeLists.txt [hunk 1]
```diff
@@ -79,6 +85,24 @@ set(MODULE_SRC
     ${CMAKE_CURRENT_LIST_DIR}/internal/msczmetareader.h
     ${CMAKE_CURRENT_LIST_DIR}/internal/notationplayback.cpp
     ${CMAKE_CURRENT_LIST_DIR}/internal/notationplayback.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/notationpaintview.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/view/notationpaintview.h
+    ${CMAKE_CURRENT_LIST_DIR}/view/notationviewinputcontroller.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/view/notationviewinputcontroller.h
```
Comment (Eism) [inline mu4/notation/CMakeLists.txt:91]: Can this class be placed in internal?

**Review by psmokotnin — APPROVED** [review]

File: mu4/notation/CMakeLists.txt (hunk 1 above)
Comment (igorkorsukov) [reply mu4/notation/CMakeLists.txt:91]: it is logical to view related, so it is better that he was in view

**Review by Eism — APPROVED** [review]

**Comment by Howard-C** [comment]
My #6443 is still not merged and it probably will have a lot of merge conflicts with this one... I'm kind of busy right now so would you do me a favour of merging that PR first? It'd help a lot, thanks!

**Comment by igorkorsukov** [comment]
> My #6443 is still not merged and it probably will have a lot of merge conflicts with this one... I'm kind of busy right now so would you do me a favour of merging that PR first? It'd help a lot, thanks!
> 
> EDIT: I saw it, thanks!! Sorry for the conflicts you have to resolve now :-)

Thank you :)
Your PR has been looked at for a long time, we should have merged it earlier

**Comment by IsaacWeiss** [comment]
So what exactly are we dealing with here? My PR #6365 has 14 conflicts with deleted or moved files, plus an undetermined number of new files that are now in the wrong directories. What changed here other than file locations? What files were deleted (I see +669 lines but -1113)?  Did anything change *within* files while they were being moved, like in PR #6372?

**Comment by igorkorsukov** [comment]
> So what exactly are we dealing with here? My PR #6365 has 14 conflicts with deleted or moved files, plus an undetermined number of new files that are now in the wrong directories. What changed here other than file locations? What files were deleted (I see +669 lines but -1113)? Did anything change _within_ files while they were being moved, like in PR #6372?

In this PR, only files have been moved, unnecessary namespaces have been removed and the header guards have been changed.
It shouldn't be difficult to correct conflicts.
Sorry for the conflicts.

**Pull request merged**